apps/client/src/app/services/import-transactions.service.ts

```diff
@@ -4,14 +4,16 @@ import { CreateOrderDto } from '@ghostfolio/api/app/order/create-order.dto';
 import { DataSource, Type } from '@prisma/client';
 import { parse } from 'date-fns';
 import { isNumber } from 'lodash';
-import { parse as csvToJson } from 'papaparse';
+import { Papa } from 'ngx-papaparse';
```

Comment:
What is the benefit of using `ngx-papaparse` over `papaparse`?

I would prefer to stick with `papaparse`, because it can be upgraded independently from _Angular_.

Reply:
changed